### PR TITLE
Fix account enumeration in 'forgot password'

### PR DESCRIPTION
**Pull request by Paul-Cl-ark**
#### Description of what you did:

- Both functions related to forgotten passwords return ok status if used not found. 
- Removed error codes as they were no longer needed
- Didn't update front end code as the message still makes sense
- Couldn't find related tests
- Couldn't find a related issue

#### Info
I think it's ready to merge but I wasn't able to test following the [contribution docs](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md) as it's not clear how to run the backend, any help doing this would be appreciated

### CODE REVIEW

File: packages/strapi-admin/controllers/Auth.js [hunk 1]
```diff
@@ -300,30 +300,20 @@ module.exports = {
...
-<p>But don’t worry! You can use the following link to reset your password:</p>
-
-<p>${url}?code=${resetPasswordToken}</p>
-
-<p>Thanks.</p>`,
+      message: `
+        <p>We heard that you lost your password. Sorry about that!</p>
+        <p>But don’t worry! You can use the following link to reset your password:</p>
+        <p>${url}?code=${resetPasswordToken}</p>
+        <p>Thanks.</p>
```
Comment (MattieBelt) [inline packages/strapi-admin/controllers/Auth.js:315]:
```suggestion

        <p>Thanks.</p>
```

**Comment by Paul-Cl-ark** [comment]
any chance on getting a review before there are too many conflicts?

**Comment by alexandrebodin** [comment]
Hi the file packages/strapi-admin/controllers/Auth.js doesn't exist in strapi anymore. I think you will need to make a new PR as we changed quite a lot of things in strapi-admin for the v3.1 :) If you create a new PR can you please close this one :)

**Comment by Paul-Cl-ark** [comment]
Yeah... 
Would have been nice to review this in time. I don't have time to make another PR right now and the code seems to have become a lot more complex/unclear so I'm going to close this PR.

**Pull request closed without merging**